### PR TITLE
Ckan ui#230

### PR DESCRIPTION
**Pull request by BrandonSharratt**

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ app.use('/api/solr', solrRouter);
 app.use('/api/resource', resourceRouter);
 app.use('/api/ckan', ckanRouter);
 app.use('/api/pow', powRouter);
-app.use('/api/analytics', analyticsRouter);
+app.use('/api/an', analyticsRouter);
```
Comment (adodge-bcg) [inline backend/app.js:102]: Was there a reason for changing this? (an is not a very clear naming)

File: backend/routes/ckan/datasets.js [hunk 2]
```diff
@@ -1,7 +1,36 @@
...
+    function formatDatasetBody(body){
+        let convertBack = false;
+        if (body.dates){
+            if (typeof(body.dates) === "string"){
+                convertBack = true;
+                body.dates = JSON.parse(body.dates);
+            }
+            console.log(body.dates);
+            for (let i=0; i<body.dates.length; i++){
+                console.log(body.dates[i]);
```
Comment (adodge-bcg) [inline backend/routes/ckan/datasets.js:16]: Were these console.logs left intentionally or should they be removed?

File: frontend/src/services/analytics.js [hunk 3]
```diff
@@ -4,18 +4,30 @@ export class Analytics {
     constructor(){}
 
     ga(){
-        const url = '/api/analytics/ga'
-        return axios.get(url, {withCredentials: true}).then(response => response.data)
+        const url = '/api/an/ga'
+        try{
+            return axios.get(url, {withCredentials: true}).then(response => response.data)
+        }catch(ex){
+            return ''
```
Comment (adodge-bcg) [inline frontend/src/services/analytics.js:11]: Do these need to return an actual error?

File: frontend/src/store/modules/nav.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    },
+}
+
+export default {
+    namespaced: true,
+    state,
+    getters,
+    actions,
+    mutations
+}
```
Comment (adodge-bcg) [inline frontend/src/store/modules/nav.js:40]:
```suggestion
}

```

File: frontend/src/components/form/DynamicForm.vue [hunk 5]
```diff
@@ -1,266 +1,268 @@
...
+                        :name="field.field_name"
+                        :value="values[field.field_name]"
+                        :label="field.label"
+                        :editing="editing"
+                        :placeholder="field.form_placeholder"
+                        :field="field"
+                        :scope="scope"
+                        :disabled="disabled"
+                        @edited="(newValue) => { updateValues(field.field_name, newValue) }"
+                        >
```
Comment (adodge-bcg) [inline frontend/src/components/form/DynamicForm.vue:16]:
```suggestion
                    >
```

File: frontend/src/components/form/DynamicForm.vue [hunk 6]
```diff
@@ -1,266 +1,268 @@
...
+                        :value="values[field.field_name]"
+                        :label="field.label"
+                        :editing="editing"
+                        :field="field"
+                        :scope="scope"
+                        trueDisplay="yes"
+                        falseDisplay="no"
+                        :disabled="disabled"
+                        @edited="(newValue) => { updateValues(field.field_name, newValue) }"
+                        >
```
Comment (adodge-bcg) [inline frontend/src/components/form/DynamicForm.vue:260]:
```suggestion
                    >
```

**Review by adodge-bcg — DISMISSED** [review]
Looks good, just some questions and suggestions

File: backend/app.js (hunk 1 above)
Comment (BrandonSharratt) [reply backend/app.js:102]: If you leave it as analytics it gets blocked by a specific analytics blocker (uBlock Origin)

File: frontend/src/services/analytics.js (hunk 3 above)
Comment (BrandonSharratt) [reply frontend/src/services/analytics.js:11]: No it only errors when it's been blocked by an analytics blocker this stops it from crashing, and no one really cares if the tracking failed.

**Review by adodge-bcg — APPROVED** [review]

**Pull request merged**